documentation/src/main/asciidoc/repositories/Repositories.adoc

```diff
@@ -16,6 +16,7 @@ public class Book {
     @Basic(optional = false)
     String title;
 
+    @Basic(optional = false)
```

Comment:
Unclear on why you want to add this annotation.

Reply:
for consistency reason. Seems `LocalDate` is in scope of `@Basic`? Anyway, if you are unclear, let me revert it back then.

Reply:
Not sure what you mean by "in scope of" here.